tests/test_reader.py

```diff
@@ -144,15 +144,14 @@ def test_reader_download_fail(local_remote_dir: Any, missing_file: str, seed: in
         os.remove(os.path.join(remote_dir, 'index.json'))
 
     # Build and iterate over a StreamingDataset
-    with pytest.raises(FileNotFoundError) as exc_info:
+    with pytest.raises(RuntimeError):
         dataset = StreamingDataset(local=local_dir,
                                    remote=remote_dir,
                                    shuffle=False,
                                    download_timeout=1,
                                    shuffle_seed=seed)
         for _ in dataset:
             pass
-    assert exc_info.match(r'.*No such file or directory*')
```

Comment:
Is there any reason why we are not validating the error message?

Reply:
Fixed